bluepysnap/morph.py

```diff
@@ -68,16 +68,33 @@ def get_morphology_dir(self, extension):
 
         return morph_dir
 
+    def get_morphology_dir(self, extension="swc"):
+        """Return morphology directory based on a given extension."""
+        morph_dir = self._get_morphology_base(extension)
+
+        if extension == "h5" and Path(morph_dir).is_file():
+            raise BluepySnapError(
+                f"'{morph_dir}' is a morphology container, so a directory does not exist"
+            )
```

Comment:
If the circuit is using a h5 container instead of a directory, I wonder if we want to raise an error anytime that `get_morphology_dir` or `get_filepath` are called, regardless of the extension specified as a parameter?
Probably in the medium/long term those functions could be deprecated, but need to check in what use cases they are needed at the moment.

Reply:
You mean if it's the only `alternaive-morphology`?

Reply:
Yeah, but after checking better it's already good as is, because the container file is retrieved only with extension=h5